homeassistant/components/huawei_lte/device_tracker.py

```diff
@@ -8,11 +8,11 @@
 
 from stringcase import snakecase
 
-from homeassistant.components.device_tracker.config_entry import ScannerEntity
-from homeassistant.components.device_tracker.const import (
+from homeassistant.components.device_tracker import (
     DOMAIN as DEVICE_TRACKER_DOMAIN,
     SourceType,
 )
+from homeassistant.components.device_tracker.config_entry import ScannerEntity
```

Comment:
Should the `ScannerEntity` and `TrackerEnitity` be exposed in `device_tracker.__init__`?

Reply:
Yes. Anything that another integration will need should be available via root.

Reply:
@balloob / @cdce8p I will start work on these as soon as #78014 is merged.
We just need to get these three over the line first:
- #78100
- #78342
- #78343